server/src/main/java/com/defold/extender/Extender.java

```diff
@@ -981,12 +1083,17 @@ private List<File> linkEngine(List<String> symbols, Map<String, Object> manifest
 
         for (String template : commands) {
             String command = templateExecutor.execute(template, context);
+            if (resolvedPods != null) {
+                command = command.replace("-miphoneos-version-min=9.0", "-miphoneos-version-min=" + resolvedPods.platformVersion);
+            }
 
             // WINE->clang transition pt2: Replace any redundant ".lib.lib"
             command = command.replace(".lib.lib", ".lib").replace(".Lib.lib", ".lib").replace(".LIB.lib", ".lib");
 
+            LOGGER.info("Linking engine - linking command {}", command);
```

Comment:
do we wanna keep these detail in logs or you just forgot to remove it?

Reply:
I forgot to remove them.